providers/mongo/pyproject.toml

```
@@ -62,7 +62,7 @@ dependencies = [
     # pymongo 4.11 had incompatible change leading to
     # TypeError: BulkOperationBuilder.add_replace() got an unexpected keyword argument 'sort'
     # See https://github.com/apache/airflow/issues/46215
-    "pymongo>=4.0.0,!=4.11,!=4.11.1",
+    "pymongo>=4.0.0,!=4.11,!=4.11.1,!=4.11.2",
```

```suggestion
    "pymongo>=4.0.0,<=4.11",
```

Let's just be defensive until we fix it instead of letting each release break CI again.

---

Hmm maybe we should simply fix it and not exclude pymongo. That one seems like an easy fix,

---

Ah no. The issue is actually with `mongomock` https://github.com/mongomock/mongomock/issues/912 and they are loking for maintaienrs: https://github.com/mongomock/mongomock/issues/914 (apparently someone volunteered 2 weeks ago). So yeah. let's exclude and link to that issue.

---

I will open new PR as @vincbeck is likely sleeping now :)

---

Good guess :)